app.json

```diff
@@ -12,10 +12,10 @@
     "ATTACHMENT_SCANNER_API_TOKEN": {
       "required": true
     },
-    "ATTACHMENT_SCANNER_URL": {
+    "ATTACHMENT_SCANNER_ENABLED": {
```

Comment:
Was there any reason to re-order these?

Reply:
It's in the commit message :trollface: 
Alphabetised, so that when you are cross referencing to the alphabetical list in the Heroku dashboard you don't have to scan the whole file for each thing.

Reply:
Heh, I only read the commit summaries, not the longer descriptions.

Good idea!